migrations/V20190830093000__create_fraud_schema.sql

```diff
@@ -0,0 +1,64 @@
+CREATE SCHEMA fraud AUTHORIZATION event_system_owner;
```

Comment:
I think I might want to call this schema `idp_data` as what it uniquely owns is the monthly IDP data – we already have fraud data elsewhere and that might be confusing. Willing to be persuaded tho.

Reply:
I'm easy ... will change it!

Reply:
Commit pushed